agent/agent.go

```diff
@@ -493,10 +493,13 @@ func (agent *Agent) Handle(cmd *proto.Cmd) *proto.Reply {
 		*/
 		agent.client.Disconnect()
 		return nil // no reply
+	// @obsolete by query/plugin/os/summary
```

Comment:
Maybe we could use `TODO` for that? Collecting those markers is supported by default in many editors and IDEs.